pkgs/games/unciv/default.nix

```diff
@@ -0,0 +1,57 @@
+{ stdenv
+, fetchurl
+, makeDesktopItem
+, makeWrapper
+, jre
+, libpulseaudio
+, libXxf86vm
+}:
+let
+  desktopItem = makeDesktopItem {
+    name = "unciv";
+    exec = "unciv";
+    comment = "Unciv, an open-source Android/Desktop remake of Civ V";
+    desktopName = "Unciv";
+    categories = "Game;";
+  };
+
+  envLibPath = stdenv.lib.makeLibraryPath [
+    libpulseaudio
+    libXxf86vm
+  ];
+
+in
+stdenv.mkDerivation rec {
+  pname = "unciv";
+
+  version = "3.9.4";
+
+  src = fetchurl {
+    url = "https://github.com/yairm210/Unciv/releases/download/${version}/Unciv.jar";
```

Comment:
Would be nice to build this from source (if doable with reasonable effort)

Reply:
Nixpkgs' Java source build support is awful, some packages succeed in doing this, but it's never pretty.